package.json

```
@@ -13,11 +13,14 @@
   "license": "CC BY-SA 4.0",
   "author": "Open Terms Archive",
   "scripts": {
-    "test": "npm run lint:markdown",
-    "lint:markdown": "markdownlint *.md content/**/*.md"
+    "test": "npm run lint:markdown && npm run lint:css",
+    "lint:markdown": "markdownlint *.md content/**/*.md",
+    "lint:css": "./node_modules/.bin/stylelint \"themes/opentermsarchive/assets/css/*.css\""
```

I'm surprised you need to explicitly use `./node_modules/.bin/`. NPM scripts should also wrap bin location. Did you try simply using:
```suggestion
    "lint:css": "stylelint themes/opentermsarchive/assets/css/*.css"
```
? 🙂

---

It's not necessary to explicity use `./node_modules/.bin/`. I thought it was a good practice, I correct